src/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import ListContainer from './ListContainer';
+import ListItemComponent from './ListItemComponent';
+
+const redditAPI = 'https://www.reddit.com/r/';
+
+export default class SubredditPosts extends Component {
```

Comment:
I also renamed the exported Class. This is to match your actual component name, and normally consumers would name their import the same name as the package they installed.

I understand that in plain English you are actually getting the sub-reddit from reddit posts, but from the user of this component's view, I think this would be clearer.